Components/Card/card.css

```diff
@@ -1,195 +1,259 @@
 @import url("../../main.css");
 
 .container {
-    max-width: var(--width-xxxs);
-    display: flex;
-    flex-direction: column;
-    flex-wrap: wrap;
-    margin: var(--space-sm);
-    box-shadow: RGBA(0, 0, 0, 0.24) 0px 3px 8px;
-    border-radius: var(--space-xxs);
+  max-width: var(--width-xxxs);
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  margin: var(--space-sm);
+  box-shadow: RGBA(0, 0, 0, 0.24) 0px 3px 8px;
+  border-radius: var(--space-xxs);
 }
 
 .content {
-    display: flex;
-    flex-direction: column;
-    flex-wrap: wrap;
-    position: relative;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  position: relative;
 }
 
 .content-row {
-    display: flex;
-    position: relative;
+  display: flex;
+  position: relative;
 }
 
 /* titles */
 
 .title {
-    padding: var(--space-xs);
+  padding: var(--space-xs);
 }
 
 .title-top {
-    background-color: var(--primary-color);
-    color: white;
-    padding: var(--space-xs);
+  background-color: var(--primary-color);
+  color: white;
+  padding: var(--space-xs);
 }
 
 .title-overlay-bottom {
-    position: absolute;
-    bottom: 0%;
-    width: 100%;
-    color: white;
-    padding: var(--space-xs);
+  position: absolute;
+  bottom: 0%;
+  width: 100%;
+  color: white;
+  padding: var(--space-xs);
 }
 
-.title-overlay-bottom>i {
-    position: absolute;
-    left: 90%;
-    bottom: 0%;
-    font-size: var(--header2);
-    color: white;
-    opacity: 0.5;
+.title-overlay-bottom > i {
+  position: absolute;
+  left: 90%;
+  bottom: 0%;
+  font-size: var(--header2);
+  color: white;
+  opacity: 0.5;
 }
 
-.title-overlay-bottom>i:hover{
-    opacity: 1;
+.title-overlay-bottom > i:hover {
+  opacity: 1;
 }
 
 .title-overlay-mid {
-    position: absolute;
-    bottom: 40%;
-    text-align: center;
-    width: 100%;
-    color: white;
+  position: absolute;
+  bottom: 40%;
+  text-align: center;
+  width: 100%;
+  color: white;
```

Comment:
You can add variables for colors which you are using multiple times rest everything looks great